tests/main.py

```diff
@@ -89,4 +89,5 @@ def run_additional_checks(argv=None):
             setattr(sphinxlint.check_line_too_long, 'enabled', True)
             setattr(checkers.rst_style.check_early_line_breaks, 'enabled', True)
             ADDITIONAL_CHECKERS.extend([checkers.resource_files.check_image_color_depth])
+            run_additional_checks(sys.argv[1:])
```

Comment:
This could be just
```suggestion
            run_additional_checks()
```

Reply:
Seems like it was accidentally removed with https://github.com/odoo/documentation/pull/4870/files#diff-b2ea600a6b04be0c003df4f4218d5437ac691b28f3f99d9fd49c13cbb0cf17aeL78